[2022-10-29 14:30:10,918] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.load_group_log scheduled__2022-10-28T14:30:00+00:00 [queued]>
[2022-10-29 14:30:10,927] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.load_group_log scheduled__2022-10-28T14:30:00+00:00 [queued]>
[2022-10-29 14:30:10,928] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-29 14:30:10,928] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-29 14:30:10,929] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-29 14:30:10,942] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): load_group_log> on 2022-10-28 14:30:00+00:00
[2022-10-29 14:30:10,945] {standard_task_runner.py:52} INFO - Started process 15585 to run task
[2022-10-29 14:30:10,948] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag', 'load_group_log', 'scheduled__2022-10-28T14:30:00+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/extract_and_load.py', '--cfg-path', '/tmp/tmpn7c6q52k', '--error-file', '/tmp/tmp30oyfo9g']
[2022-10-29 14:30:10,949] {standard_task_runner.py:80} INFO - Job 175: Subtask load_group_log
[2022-10-29 14:30:11,010] {task_command.py:369} INFO - Running <TaskInstance: dag.load_group_log scheduled__2022-10-28T14:30:00+00:00 [running]> on host cf01c04ee00d
[2022-10-29 14:30:11,086] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag
AIRFLOW_CTX_TASK_ID=load_group_log
AIRFLOW_CTX_EXECUTION_DATE=2022-10-28T14:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-28T14:30:00+00:00
[2022-10-29 14:30:16,319] {python.py:173} INFO - Done. Returned value was: None
[2022-10-29 14:30:16,335] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dag, task_id=load_group_log, execution_date=20221028T143000, start_date=20221029T143010, end_date=20221029T143016
[2022-10-29 14:30:16,392] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-29 14:30:16,424] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
